assets/src/date/index.js

```diff
@@ -23,15 +23,13 @@
  * @param {*} dateSettings
  */
 
-export const DEFAULT_DATE_SETTINGS = {
-  dateFormat: 'Y-m-d',
-  timeFormat: 'g:i a',
-  gmtOffset: null,
-  timezone: '',
-};
+export { subMinutes, isValid, differenceInSeconds } from 'date-fns';
+export { toDate } from 'date-fns-tz';
 
-export { isToday } from './isToday';
-export { isYesterday } from './isYesterday';
-export { getDateObjectWithTimezone } from './getDateObjectWithTimezone';
-export { getTimeFromNow } from './getTimeFromNow';
-export { getRelativeDisplayDate } from './getRelativeDisplayDate';
+export { getSettings, updateSettings } from './settings';
+export { default as format } from './format';
+export { default as getRelativeDisplayDate } from './getRelativeDisplayDate';
+export { default as formatDate } from './formatDate';
+export { default as formatTime } from './formatTime';
```

Comment:
I don't know how I feel about all these functions being _impure_ (stateful), but I guess it's the easiest way to do this without having to send the same formatting parameters everywhere.

Reply:
It's certainly not the best approach, but I think once this is merged things are in a better shape should there be a need for further refactoring.